docs/docs/img/UseCase.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The Forms feed into the ACCS which then handles the HTLCs.
Also it looks like Bitcoin is always proposer.
We do not have a login.

-> teams.